website/static/js/citations.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+var $ = require('jquery');
+require('select2');
+
+var r = function(query) {
+    query.callback({results: [
+        {
+            _id: "academy-of-management-review",
+            summary: null,
+            short_title: "AMR",
+            title: "Academy of Management Review"
+        }
+    ]})
+}
+
+var formatResult = function(state) {
+    var html = "<div class='citation-result-title'>" + state.title + "</div>";
+    //if (state.short_title_!== null) {
+    //    html += "<div class='citation-result-slug'>" + state.short_title + "</div>";
+    //};
+    //if (state.summary !== null) {
+    //    html += "<div class='citation-result-summary'>" + state.summary + "</div>";
+    //};
+    return html;
+};
+
+var formatSelection = function(state) {
+    return state.title;
+};
+
+var input = $('#citation-style-input');
+var citationElement = $('#citation-text');
+
+input.select2({
```

Comment:
Can this module be refactored to get rid of import side effects? For example, all this initialization can go in a constructor:

```js
function CitationWidget(inputSelector, textSelector) {
    // set up select2, etc.
}
```

Then in `project-dashboard-page.js`

```javascript
var CitationWidget = require('../citations.js');
new CitationWidget('#citationStyleInput, '#citationText');
```